app/uk/gov/hmrc/trusts/models/TrustRegistrationModels.scala

```diff
@@ -395,41 +437,54 @@ object NonUKType {
 
 case class PropertyLandType(buildingLandName: Option[String],
                             address: Option[AddressType],
-                            valueFull: Long,
-                            valuePrevious: Long)
+                            valueFull: Option[Long],        // now optional with 5MLD
+                            valuePrevious: Option[Long])    // now optional with 5MLD
 
 object PropertyLandType {
   implicit val propertyLandTypeFormat: Format[PropertyLandType] = Json.format[PropertyLandType]
 }
 
 case class BusinessAssetType(orgName: String,
                              businessDescription: Option[String],
-                             address: AddressType,
-                             businessValue: Long)
+                             address: Option[AddressType],  // now optional with 5MLD
+                             businessValue: Option[Long])   // now optional with 5MLD
 
 object BusinessAssetType {
   implicit val businessAssetTypeFormat: Format[BusinessAssetType] = Json.format[BusinessAssetType]
 }
 
 case class OtherAssetType(description: String,
-                          value: Long)
+                          value: Option[Long])    // now optional with 5MLD
```

Comment:
OtherAsset value should be mandatory as if they have one FE will always collect at registration